web/modules/custom/feedback/src/Form/FeedbackEntityForm.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\feedback\Form;
+
+use Drupal\Component\Datetime\TimeInterface;
+use Drupal\Core\Entity\ContentEntityForm;
+use Drupal\Core\Entity\EntityRepositoryInterface;
+use Drupal\Core\Entity\EntityTypeBundleInfoInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Form controller for the feedback entity edit forms.
+ *
+ * @ingroup feedback
+ */
+class FeedbackEntityForm extends ContentEntityForm {
+
+  /**
+   * Gets the current active user.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  protected $currentUser;
+
+  /**
+   * FeedbackForm constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityRepositoryInterface $entity_repository
+   *   The entity repository.
+   * @param \Drupal\Core\Entity\EntityTypeBundleInfoInterface $entity_type_bundle_info
+   *   The entity type bundle service.
+   * @param \Drupal\Component\Datetime\TimeInterface $time
+   *   The time service.
+   * @param \Drupal\Core\Session\AccountProxyInterface $current_user
+   *   Gets the current active user.
+   */
+  public function __construct(EntityRepositoryInterface $entity_repository, EntityTypeBundleInfoInterface $entity_type_bundle_info = NULL, TimeInterface $time = NULL, AccountProxyInterface $current_user) {
+    parent::__construct($entity_repository, $entity_type_bundle_info, $time);
+    $this->currentUser = $current_user;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('entity.repository'),
+      $container->get('entity_type.bundle.info'),
+      $container->get('datetime.time'),
+      $container->get('current_user')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'feedback_entity_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    /* @var $entity \Drupal\feedback\Entity\Feedback */
+    $form = parent::buildForm($form, $form_state);
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function save(array $form, FormStateInterface $form_state) {
+    // Get current user's id.
+    $user_id = $this->currentUser->id();
+    if ($user_id == 1) {
+      $form_state->setRedirect('feedback.report');
+      drupal_set_message($this->t('The message has been changed'));
+    }
+    else {
+      $form_state->setRedirect('entity.feedback.add_entity');
+      drupal_set_message($this->t('Your message has been sent'));
+    }
```

Comment:
I did it in such a way that admin should not send the messages to himself. He just changes them if there is a need.